components/job-orchestration/job_orchestration/executor/compress/compression_task.py

```
@@ -68,19 +70,20 @@
     db_cursor.executemany(
         f"""
-        INSERT INTO {table_prefix}{ARCHIVE_TAGS_TABLE_SUFFIX} (archive_id, tag_id)
+        INSERT INTO {archive_tags_table_name} (archive_id, tag_id)
         VALUES (%s, %s)
         """,
         [(archive_id, tag_id) for tag_id in tag_ids],
     )
 
 
-def update_job_metadata_and_tags(db_cursor, job_id, table_prefix, tag_ids, archive_stats):
+def update_job_metadata_and_tags(db_cursor, job_id, table_prefix, dataset, tag_ids, archive_stats):
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)